proto/desmos/tokenfactory/v1beta1/params.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package desmos.tokenfactory.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v5/x/tokenfactory/types";
+
+// Params defines the parameters for the tokenfactory module.
+//
+// Since: Desmos 6.0.0
+message Params {
```

Comment:
Having a custom tokenfactory params can make our proto does not depend on osmosis protos